### DISCUSSION TITLE
Oh-my-posh dont work in PS 5.1 and PS 7.3.3 simultaneously

### DISCUSSION THREAD

**Discussion by ikocs** [JanDeDobbeleer/oh-my-posh#3712] [Troubleshoot] [answered]
I want to use oh-my-posh on PowerShell 7.3.3. In PS 5.1 all work greate, but in PS 7.3.3 I get with string
```shell
. $PROFILE
(@(& '~/AppData/Local/Programs/oh-my-posh/bin/oh-my-posh.exe' init pwsh --config='~\AppData\Local\oh-my-posh\config.omp.json' --print) -join "`n") | Invoke-Expression
```

or:

```shell
oh-my-posh init pwsh --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/atomic.omp.json'
> (@(& '~/AppData/Local/Programs/oh-my-posh/bin/oh-my-posh.exe' init pwsh --config='~\AppData\Local\oh-my-posh\config.omp.json' --print) -join "`n") | Invoke-Expression
```

I'm new, don't be too hard on me)

**Comment by JanDeDobbeleer** [comment, marked as answer]
The initialization is the same for both, all you need is `oh-my-posh init pwsh | Invoke-Expression` in both shells (and yes, you can specify the config flag like you did). Do know they both have their own PROFILE. Adding it to 1 doesn't make it work on the other, you need to adjust both.

**Reply by ikocs** [reply]
I figured out different profile files and it all worked. Thank you!
